vipps-qr-api.md

```diff
@@ -209,6 +213,31 @@ The response will be similar to this, where the URL in the responseBody will be
 
 **Please note:** The `expiresIn` value is in seconds.
 
+### Polling 
+On OTP qrs, merchants will need to [poll](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#polling-guidelines) to get the result of the payment. In physical context, we recommend a polling interval of 1 second. Once a customer has scanned the QR, it is possible for merchants to show the customer a "waiting/spinner"-screen while they finish the payment. This will feel comforting for the user as they will get feedback that the payment is underway.
```

Comment:
```suggestion
On OTP QR codes, merchants will need to [poll](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#polling-guidelines) to get the result of the payment. In a physical context, we recommend a polling interval of one second. Once a customer has scanned the QR code, it is possible for merchants to show the customer a "waiting/spinner"-screen while they finish the payment. This will feel comforting for the user as they will get feedback that the payment is underway.
```